README.md

```diff
@@ -145,6 +152,119 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 1. Node.js Client API Overview
+
+In the API, most of the functions return `Promise`. You need to be able to use promises to use the client. Promises provide a better way of working with callbacks. You can chain asynchronous functions by `then()` function of promise. Also, you can use `async/await`, if you use Node v8 and higher. If you are not familiar with promises, please first learn how to use them. If you are ready to go, let's start to use Hazelcast Node.js Client!
```

Comment:
`If you are not familiar with promises, please first learn how to use them.` sounds kind of aggressive to me. I am not objecting it, just making a note. I leave the language to @Serdaro

Reply:
My suggestion for the whole paragraph:

Most of the functions in the API return `Promise`. Therefore, you need to be familiar with the concept of promises to use the Node.js client. If not, you can learn about them using various online resources. 

Promises provide a better way of working with callbacks. You can chain asynchronous functions by `then()` function of promise. Also, you can use `async/await`, if you use Node.js 8 and higher versions. 

If you are ready to go, let's start to use Hazelcast Node.js client!